src/qt/rpcconsole.h

```diff
@@ -136,6 +136,11 @@ public Q_SLOTS:
     void cmdRequest(const QString &command, const WalletModel* wallet_model);
 
 private:
+    struct TranslatedStrings {
```

Comment:
Interesting technique. I guess translators will see `TranslatedStrings` as location, not `rpcconsole.h`? But I guess that doesn't matter, not much context needed?

Reply:
I've tried with some small test project, and `lupdate`/`linguist` uses parent class as context, so no issues here.

Reply:
@Talkless thank you for checking!